### DISCUSSION TITLE
Free Public Server List Updating In-App

### DISCUSSION THREAD

**Discussion by trymeouteh** [rustdesk/rustdesk#6862] [Ideas]
I would like to suggest the following feature within the RustDesk app.

- A way for RustDesk app to fetch the latest list of public servers available to use
- Have RustDesk app check for updates to this list in thd app everytime RustDesk launches and every X hours
- Always have RustDesk choose the best free server by default with low ping (If a server is not setup)

Such a feature will make it easier to setup for your grandma. This way grandma will not need to know how to add or update the free pubic server in the app. And this should be the default setup of RustDesk and can be changed by advanced users.

**Comment by pdifeo** [comment]
As I understand it, the relay server chosen is the one that is geographically closest and not the one with the lowest latency. My opinion is that this is not the best strategy for choosing a relay server, as the one chosen could be the one closest, but with worse connectivity between the two peers
